example/main.tf

```diff
@@ -5,7 +5,6 @@ provider "equinix" {
 
 module "pa-vm" {
   source               = "equinix/pa-vm/equinix"
```

Comment:
This `source` needs to be changed to point to the root module; we should also include commented-out lines that show how to pull down the published module.  Here's an example of what that looks like in the CPEM example of the `equinix-labs/terraform-equinix-kubernetes-cluster` module: https://github.com/equinix-labs/terraform-equinix-kubernetes-cluster/blob/main/examples/cpem-add-on/main.tf#L2C3-L4